examples/bzlmod/MODULE.bazel

```diff
@@ -1,9 +1,9 @@
 module(
     name = "rules_jvm_external_examples_bzlmod",
-    version = "5.0.0",
+    version = "4.2.2",
```

Comment:
extract to constant?

Reply:
I didn't think a `MODULE.bazel` file could `load` an external file: I believe that they need to be entirely self-contained.